.gitignore

```diff
@@ -158,3 +158,4 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+city_temperature.db
```

Comment:
The `city_temperature.db` file is listed here. Ensure that this database file is not needed for other developers or in production. If it's a local development database, it's fine to ignore it. Otherwise, consider whether it should be included in version control or if there are other means to share it.